viewpump/src/main/java/io/github/inflationx/viewpump/ViewPumpLayoutInflater.java

```diff
@@ -0,0 +1,436 @@
+            // Set the constructor ars up for the createView, not sure why we can't pass these in.
+            mConstructorArgsArr[0] = viewContext;
+            ReflectionUtils.setValue(mConstructorArgs, this, mConstructorArgsArr);
+            try {
+                view = createView(name, null, attrs);
+            } catch (ClassNotFoundException ignored) {
+            } finally {
+                mConstructorArgsArr[0] = lastContext;
+                ReflectionUtils.setValue(mConstructorArgs, this, mConstructorArgsArr);
+            }
+        }
+        return view;
+    }
+
+    @TargetApi(Build.VERSION_CODES.HONEYCOMB)
+    private View superOnCreateView(View parent, String name, AttributeSet attrs) {
+        try {
+            return super.onCreateView(parent, name, attrs);
+        } catch (ClassNotFoundException e) {
+            return null;
+        }
+    }
+
+    private View superOnCreateView(String name, AttributeSet attrs) {
+        try {
+            return super.onCreateView(name, attrs);
+        } catch (ClassNotFoundException e) {
+            return null;
+        }
+    }
+
+    // ===
+    // View creators
+    // ===
+
+    private static class ActivityViewCreator implements FallbackViewCreator {
+        private final ViewPumpLayoutInflater inflater;
+        private final View view;
+
+        public ActivityViewCreator(ViewPumpLayoutInflater inflater, View view) {
+            this.inflater = inflater;
+            this.view = view;
+        }
+
+        @Override
+        public View onCreateView(View parent, String name, Context context, AttributeSet attrs) {
+            return inflater.createCustomViewInternal(parent, view, name, context, attrs);
+        }
+    }
+
+    private static class ParentAndNameAndAttrsViewCreator implements FallbackViewCreator {
+        private final ViewPumpLayoutInflater inflater;
+
+        public ParentAndNameAndAttrsViewCreator(ViewPumpLayoutInflater inflater) {
+            this.inflater = inflater;
+        }
+
+        @Override
+        public View onCreateView(View parent, String name, Context context, AttributeSet attrs) {
+            return inflater.superOnCreateView(parent, name, attrs);
+        }
+    }
+
+    private static class NameAndAttrsViewCreator implements FallbackViewCreator {
+        private final ViewPumpLayoutInflater inflater;
+
+        public NameAndAttrsViewCreator(ViewPumpLayoutInflater inflater) {
+            this.inflater = inflater;
+        }
+
+        @Override
+        public View onCreateView(View parent, String name, Context context, AttributeSet attrs) {
+            // This mimics the {@code PhoneLayoutInflater} in the way it tries to inflate the base
+            // classes, if this fails its pretty certain the app will fail at this point.
+            View view = null;
+            for (String prefix : sClassPrefixList) {
+                try {
+                    view = inflater.createView(name, prefix, attrs);
```

Comment:
this should short if `view != null` See: https://github.com/chrisjenx/Calligraphy/pull/361